doc_classes/TrainPart.xml

```diff
@@ -30,42 +30,6 @@
 				Returns dictionary ov properties with values from iternal mover
 			</description>
 		</method>
-		<method name="log">
```

Comment:
that was shortcut for logging system. It was used in C(++) and GDScript. Now you cant use this shortcut in GDScript.